### PR TITLE
feat(projects.onboarding): adapt onboarding page to trusted customers 

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `MANAGER-7953`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-8735
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
1. Add trusted PCI zone badge
2. Add text to explain why TZ customers can't create a project
3. Disable create project btn for TZ customers
4. Add a phone callback button

### CODE REVIEW

**Review by varun257 — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/onboarding/onboarding.constant.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  DEFAULT: 'https://www.ovhcloud.com/en-gb/enterprise/trusted-zone/#callback',
+  FR: 'https://www.ovhcloud.com/fr/enterprise/trusted-zone/#callback',
+  DE: 'https://www.ovhcloud.com/de/enterprise/trusted-zone/#callback',
+  ES: 'https://www.ovhcloud.com/es-es/enterprise/trusted-zone/#callback',
+  IE: 'https://www.ovhcloud.com/en-ie/enterprise/trusted-zone/#callback',
+  IT: 'https://www.ovhcloud.com/it/enterprise/trusted-zone/#callback',
+  NL: 'https://www.ovhcloud.com/nl/enterprise/trusted-zone/#callback',
+  PL: 'https://www.ovhcloud.com/pl/enterprise/trusted-zone/#callback',
+  PT: 'https://www.ovhcloud.com/pt/enterprise/trusted-zone/#callback',
+  GB: 'https://www.ovhcloud.com/en-gb/enterprise/trusted-zone/#callback',
```
Comment (radireddy) [inline packages/manager/modules/pci/src/projects/onboarding/onboarding.constant.js:11]:
```suggestion

```
GB not required. DEFAULT will take care of it.

File: packages/manager/modules/pci/src/projects/onboarding/onboarding.routing.js [hunk 2]
```diff
@@ -29,5 +19,20 @@ export default /* @ngInject */ ($stateProvider) => {
               }
             : null,
         ),
+    resolve: {
+      breadcrumb: () => null, // Hide breadcrumb
+
+      goToCustomerPhoneCallback: /* @ngInject */ (
```
Comment (radireddy) [inline packages/manager/modules/pci/src/projects/onboarding/onboarding.routing.js:25]:
```suggestion
      openTrustedZoneCallbackPage: /* @ngInject */ (
```

File: packages/manager/modules/pci/src/projects/onboarding/onboarding.constant.js (hunk 1 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/onboarding/onboarding.constant.js:11]: Yes sure, i preferred to have the dedicated entry.
I use default one for all missed others subs that could be forgot.

File: packages/manager/modules/pci/src/projects/onboarding/onboarding.routing.js (hunk 2 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/onboarding/onboarding.routing.js:25]: Yep better

**Review by JeremyDec — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/onboarding/onboarding.constant.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+export const TRUSTED_ZONE_CALLBACK_PHONE_PAGES = {
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/onboarding/onboarding.constant.js:1]: **question:** Is there any reason to name it this way? Why Phone?
cc @mohammed-zahaf

**Pull request merged**